.circleci/config.yml

```
@@ -61,7 +61,6 @@ workflows:
                 - 20.2.0
                 - 18.14.0
                 - 16.19.0
```

Perhaps we should conduct tests based on version 16.0.0.

```suggestion
                - 16.0.0
```